packages/react-app-scaffolder/app/templates/base-is-not-sass/src/scripts/webpack-prod.js

```diff
@@ -8,12 +8,16 @@ const { EnvironmentPlugin } = require('webpack')
 const { PATHS } = require('./constants')
 const path = require('path')
 
+// HEAD~1: Hash appended to files in commit N will be generated by commit N - 1
```

Comment:
I think the better one is HEAD~2, we should support the latest version and latest-1 version. It'll be more safer